web/profiles/joinup/joinup.behat.inc

```diff
@@ -68,6 +73,37 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
   }
 
   /**
+   * Creates and authenticates a user with the given og role(s).
+   *
+   * Multiple roles can be passed separated with comma.
+   *
+   * @Given I am logged in as a user with the :role role(s) of the :rdf_entity :rdf_entity_bundle
+   * @Given I am logged in as a/an :role of the :rdf_entity :rdf_entity_bundle
+   */
+  public function assertAuthenticatedByOgRole($role, $rdf_entity, $rdf_entity_bunlde) {
+    $entity = $this->getRdfEntityByLabel($rdf_entity, $rdf_entity_bunlde);
+    $roles = explode(',', $role);
+    $roles = array_map('trim', $roles);
+    $roles = $this->convertOgRoleNamesToIds($roles, $entity);
+
+    // Check if a user with this role is already logged in.
+    if (!$this->loggedInWithOgRole($roles, $entity)) {
+      $random = $this->getRandom()->name(8);
+      // Create user (and project)
+      $user = (object) array(
+        'name' => $random,
+        'pass' => $random,
+      );
+      $user->mail = "{$user->name}@example.com";
+      $this->userCreate($user);
+      $this->subscribeUserToGroup($user->uid, $entity, $roles);
+
+      // Login.
+      $this->login();
+    }
```

Comment:
This code seems to be based on `DrupalContext::assertAuthenticatedByRole()`, let's add an `@see` to the documentation so it is clear that these methods are related.